app/assets/javascripts/modules/track-click.js

```diff
@@ -10,14 +10,24 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       var options = {},
           category = element.attr('data-track-category'),
           action = element.attr('data-track-action'),
-          label = element.attr('data-track-label');
+          label = element.attr('data-track-label'),
+          dimension = element.attr('data-track-dimension'),
+          customDimension = element.data('track-custom-dimension');
 
       if (label) {
         options.label = label;
       }
 
       function trackClick() {
-        if (GOVUK.analytics && GOVUK.analytics.trackEvent) {
+        if (!GOVUK.analytics) {
+          return;
+        }
+
+        if (customDimension && dimension && GOVUK.analytics.setDimension) {
+          GOVUK.analytics.setDimension(customDimension, dimension);
```

Comment:
That's configured here:
https://github.com/alphagov/static/blob/9e8795b262c948a65b798029864cbe04f193262a/app/assets/javascripts/analytics/static-analytics.js#L26-L28

And here:
https://github.com/alphagov/govuk_frontend_toolkit/blob/master/javascripts/govuk/analytics/download-link-tracker.js

Though you're right, I can't see anything being tracked.